apps/examples/testcase/le_tc/kernel/itc_pthread.c

```diff
@@ -0,0 +1,550 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file itc_pthread.c
+/// @brief ITC Test Case Example for Pthread API
+
+/****************************************************************************
+ * Included and defines
+ ****************************************************************************/
+
+#include <pthread.h>
+#include <stdio.h>
+#include <errno.h>
+#include <pthread.h>
+#include <stdlib.h>
+#include <tinyara/pthread.h>
+#include <tinyara/config.h>
+#include "tc_internal.h"
+#include <sys/types.h>
+#include <stdbool.h>
+#include <unistd.h>
+#include <signal.h>
+#include <sched.h>
+
+#define WAIT_TIME_1                   1
+#define WAIT_TIME_2                   2
+#define WAIT_TIME_4                   4
+#define WAIT_TIME_8                   8
+#define WAIT_TIME_0                   0
+#define BUF_LEN 32
+#define INIT_MINUSONE -1
+#define SIG_ERROR 0
+#define LOOP_SIZE CONFIG_ITC_KERN_PTHREAD_LOOP_SIZE
+/****************************************************************************
+ * Helper Functions
```

Comment:
What is ```helper functions```?

Reply:
These are the thread functions which are called ,while calling pthread_create .